docs/export/export.md

```diff
@@ -0,0 +1,465 @@
+
+# Exporting and serializing staged-out computations
```

Comment:
If you add `(export)=` on the line above the heading, then you can cross reference this doc elsewhere using ``` :ref:`export` ``` in docstrings and restructured text files, or ``` {ref}`export` ``` in markdown files.

Reply:
Done, thanks!